### ISSUE TITLE
Document the use of labels

### ISSUE THREAD

**Issue by foolip** [w3c/web-platform-tests#8286]
I can't find any documentation of the labels in use in this repo. I'd like to write that documentation, and based on that work out some triage process for groups of labels.

@tobie, where is the code that automatically creates labels for each directory?

**Comment by foolip** [comment]
FWIW, these are the directory names in / and /css/ that don't have a label:
* CSS1
* XMLHttpRequest
* cors
* css
* css-block
* css-exclusions
* css-gcpm
* css-speech
* fonts
* images
* media
* tools

Seems about right, but suggests there's some special handling for XHR.

And these are the labels that don't correspond to a directory:
* DOM-parsing
* DOMEvents
* ElementTraversal
* WARNING
* animation-timing
* app-uri
* appmanifest
* assets
* assumptions
* auxclick
* chromium-export
* ci_stability
* clipboard
* css-build-system
* css-built-testsuites
* css-font-display
* css-logical-properties
* css-overflow
* css-sizing
* deferred
* difficulty:easy
* difficulty:less-easy
* do not merge yet
* duplicate
* example
* ext-xhtml-pubid
* flaky
* frame-timing
* good first issue
* help wanted
* html-templates
* http-demo
* idlharness.js
* image-decodes
* imagebitmap
* in progress
* infra
* invalid
* lint
* lint.whitelist
* manifest
* mediacapture-depth
* microdata
* mimesniff
* mpegmap
* overflow
* policy
* priority:backlog
* priority:roadmap
* priority:urgent
* progress-events
* reviewed
* runner
* safe-html-types
* scroll-anchoring
* scroll-into-view
* selectors-api
* serve
* service-worker
* servo-export
* shadow-dom-untriaged
* signature
* size:project
* stale:awaiting-merge-conflict-resolution
* stale:awaiting-review
* stale:awaiting-submitter-response
* stale:review-comments-on-critic
* status:awaiting-merge
* status:needs-changes
* status:needs-collaborator
* status:needs-owners
* status:needs-review
* status:needs-spec-decision
* test
* test-dir
* testdriver
* testharness.js
* third_party
* tmp
* touch
* ttml
* type:bug
* type:cleanup
* type:missing-coverage
* type:untestable
* typedarrays
* unspecified-css-subresources
* viewport
* wakelock
* webmidi
* webshare
* wg-annotation
* wg-audio
* wg-audiovideo_tt
* wg-css
* wg-dap
* wg-dom
* wg-fonts
* wg-geolocation
* wg-html
* wg-i18n_core
* wg-math
* wg-payments
* wg-pointerevents
* wg-secondscreen
* wg-svg
* wg-sw
* wg-sysapps
* wg-tag
* wg-testing_browser
* wg-wai_pf
* wg-web-notifications
* wg-webapps
* wg-webappsec
* wg-webauthn
* wg-webcrypto
* wg-webevents
* wg-webperf
* wg-webplatform
* wg-webrtc
* work-in-progress
* wpt
* wptrunner
* xhr

Mostly reasonable, but some stuff in need of cleanup, like WARNING and tmp. (I'll clean a bit now.)

**Comment by tobie** [comment]
https://github.com/tobie/ganesh/blob/master/lib/metadata/raw-labels.js

and 

https://github.com/tobie/ganesh/blob/master/lib/metadata/labels.js

**Comment by gsnedders** [comment]
@foolip note if someone creates a PR with a new directory a label gets created for it, so it's quite likely we'll have labels that don't correspond to a directory

**Comment by foolip** [comment]
Yep, that's a thing I'd like to document, that if you push a PR with a "test" directory, a test label will be created. People probably don't expect that :)
